homeassistant/components/hisense_aehw4a1/.translations/bg.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please remove all the translation files except the default English translation. Translations will be handled by Lokalise.

Reply:
I left only string.json